test/handler.test.js

```diff
@@ -244,6 +245,30 @@ describe('HTTP request handler', () => {
       ])
     })
   })
+
+  describe('GET /nodes/daily', () => {
```

Comment:
Could you please move the new tests into a new test file (e.g. `test/platform-routes.test.js`) so that it's easier to duplicate these tests to Voyager.